### PR TITLE
[onert/test] Set test command common variables

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit adds test command common variable "CACHE_PATH" and use common variables on commands.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Draft: #10496

### CODE REVIEW

File: tests/scripts/command/prepare-model [hunk 1]
```diff
@@ -14,9 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-COMMAND_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-INSTALL_DIR="$(dirname $(dirname $COMMAND_DIR))"
-
```
Comment (glistening) [inline tests/scripts/command/prepare-model:19]: By removing these variables and defining them in `onert_test`, `prepare-model` seems not standalone script any longer. Do you want to make `onert-test` as main driver and others as subcommands?

Comment (hseok-oh) [reply tests/scripts/command/prepare-model:19]: Right. Scripts under `command` don't have execute permission now.

**Review by glistening — APPROVED** [review]
LGTM

**Review by ragmani — APPROVED** [review]
LGTM

**Pull request merged**